packages/sources/coinmetrics/src/transport/price-ws.ts

```diff
@@ -78,9 +91,19 @@ export const handleAssetMetricsMessage = (
   message: WsAssetMetricsMessage,
 ): ProviderResult<WsTransportTypes>[] | undefined => {
   logger.trace(message, 'Got response from websocket')
+
   if ('error' in message) {
     // Is WsAssetMetricsErrorResponse
     logger.error(message, `Error response from websocket`)
+
+    const regex = /'([^']+)'/g
```

Comment:
TypeScript has a handy type for Regular Expressions, which we could use here: `const regex = new RegExp(/'([^']+)'/g)`.